device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -241,10 +241,8 @@ private void startWorkerThreads()
      * Must be called to terminate the background thread that is sending data to
      * IoT Hub. After {@code close()} is called, the IoT Hub client is no longer
      *  usable. If the client is already closed, the function shall do nothing.
-     *
-     * @throws IOException if the connection to an IoT Hub cannot be closed.
      */
-    public void close() throws IOException
```

Comment:
Same; breaking change?

Reply:
Nope. We construct this internally, but we never ask users to create an instance of this class